api/app/routers/pteams.py

```diff
@@ -587,6 +590,52 @@ def get_vuln_ids_tied_to_service_packages(
     }
 
 
+@router.get(
+    "/{pteam_id}/ticket_counts",
+    response_model=schemas.ServicePackageTicketCountsSolvedUnsolved,
+)
+def get_ticket_counts_tied_to_service_packages(
```

Comment:
```suggestion
def get_ticket_counts_tied_to_service_package(
```
packageは単数。
get_vuln_ids_tied_to_service_packages()も同様。
テストクラス名にも波及。